definitions/npm/rsuite_v3.x.x/flow_v0.53.x-/rsuite_v3.x.x.js

```diff
@@ -0,0 +1,1738 @@
+declare module "rsuite" {
+  import type {
+    Node,
+    Element,
+    ElementType,
+    Component,
+    ChildrenArray,
+    ComponentType,
+    ElementRef,
+    Ref
+  } from "react";
```

Comment:
This won't work, check the contributing guidelines for alternatives.

Reply:
Has been fixed.